src/server/game/Spells/SpellMgr.cpp

```diff
@@ -3261,26 +3261,26 @@ void SpellMgr::LoadSpellCustomAttr()
 
     CreatureAI::FillAISpellInfo();
 
-    sLog->outString(">> Loaded spell custom attributes in %u ms", GetMSTimeDiffToNow(oldMSTime));
+    sLog->outString(">> Loaded SpellInfo custom attributes in %u ms", GetMSTimeDiffToNow(oldMSTime));
     sLog->outString();
 }
 
-inline void ApplySpellFix(std::initializer_list<uint32> spellIds, void(*fix)(SpellEntry*))
+inline void ApplySpellFix(std::initializer_list<uint32> spellIds, void(*fix)(SpellInfo*))
 {
     for (uint32 spellId : spellIds)
     {
-        SpellEntry const* spellInfo = (SpellEntry*)sSpellStore.LookupEntry(spellId);
+        SpellInfo const* spellInfo = (SpellInfo*)sSpellStore.LookupEntry(spellId);
```

Comment:
also here, it's pointless to repeat `SpellInfo` twice in a row, we can replace the first `SpellInfo` with `auto`

Reply:
Build error with that